src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,199 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import subway.dto.LinePatchResponse;
+import subway.dto.LineRequest;
+import subway.utils.DatabaseCleanup;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("지하철 노선 관련 기능")
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class LineAcceptanceTest {
+
+    private static final int GANG_NAM_STATION = 1;
+    private static final int YANG_JAE_STATION = 2;
+    private static final int SEO_HYEON_STATION = 3;
+
+    @Autowired
+    private DatabaseCleanup databaseCleanup;
+
+    @LocalServerPort
+    int port;
+
+    @BeforeEach
+    public void setUp() {
+        RestAssured.port = port;
+        databaseCleanup.execute();
+        StationAcceptanceTest.createStation("강남역");
```

Comment:
```suggestion
        GANG_NAM_STATION = StationAcceptanceTest.createStation("강남역");
```

바로 초기화 해준다면 테스트가 더 정밀해지지 않을까합니다 😊